sites/aalto/aalto.yaml

```diff
@@ -6,3 +6,6 @@ A01-local: |
 B14-local: |
   `Aalto scicomp <https://scicomp.aalto.fi/data/>`__
   `Aalto RDM <https://aalto.fi/rdm>`__
```

Comment:
I notice that this is nothing - somehow it's a semi-duplicate of C14, which we talked about.  Can be removed.
```suggestion
```